tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -318,6 +318,7 @@ def __init__(self, suite_name, test_queue, test_source_cls, browser_cls, browser
         self.debug_info = debug_info
 
         self.manager_number = next_manager_number()
+        self.recording = recording
```

Comment:
Should we assert that `recording` is not `None` here, or just leave it to fire an exception when `set` is called on it?